### PR TITLE
Add quantify decoys option to FFid

### PR DESCRIPTION
**Pull request by timosachsenberg**
# Description

Added the possibility to quantify decoys in FFid (e.g., for error estimation)

# Checklist:
- [x] Make sure that you are listed in the AUTHORS file
- [x] Add relevant changes and new features to the CHANGELOG file
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] New and existing unit tests pass locally with my changes
- [x] Updated or added python bindings for changed or new classes. (Tick if no updates were necessary.)

### CODE REVIEW

**Comment by timosachsenberg** [comment]
will be rebased after other PR is merged

File: src/tests/topp/ProteomicsLFQ_5_out.csv [hunk 1]
```diff
@@ -25,6 +25,7 @@ P02769|ALBU_BOVIN,EC(Carbamidomethyl)C(Carbamidomethyl)DKPLLEK,2,NA,0,L,1,1,1,1,
 P02769|ALBU_BOVIN,EC(Carbamidomethyl)C(Carbamidomethyl)DKPLLEK,3,NA,0,L,1,1,1,1,1.240657e07,"BSA1_F1.mzML"
 P02769|ALBU_BOVIN,EC(Carbamidomethyl)C(Carbamidomethyl)DKPLLEK,3,NA,0,L,3,3,5,1,2.598459e06,"BSA3_F1.mzML"
 P02769|ALBU_BOVIN,ETYGDMADC(Carbamidomethyl)C(Carbamidomethyl)EK,2,NA,0,L,2,2,3,1,6.884387e05,"BSA2_F1.mzML"
+P02769|ALBU_BOVIN_rev,EVEAIC(Carbamidomethyl)HSK,3,NA,0,L,1,1,1,1,1.358151e06,"BSA1_F1.mzML"
```
Comment (timosachsenberg) [inline src/tests/topp/ProteomicsLFQ_5_out.csv:28]: Should we skip export to MSstats?

Comment (jpfeuffer) [reply src/tests/topp/ProteomicsLFQ_5_out.csv:28]: hmm probably, for now. Otherwise, people need to filter in the R scripts

**Pull request merged**